### PR TITLE
1425   task 7

### PR DESCRIPTION
**Pull request by zarafshanejaz786**
Ticket Link:
https://dev.azure.com/CodeNinjaConsulting/Talent%20Pool/_workitems/edit/1483

Create a new data view page and display some data in the table. The data shown must be fetched from an external API. Add a link to the data view page in the page’s header. The new page should also have the same header, with a link to go back to the portfolio page.  

This should consist of areas/concepts that are specified

Add a section to display a table with some data. Research and choose a public API that provides some data (e.g., JSONPlaceholder for mock data). 

Use the fetch API to retrieve data from the chosen API. 
Display the fetched data (e.g., posts, comments, or any relevant data) in a visually appealing table on the interactive web page.

### CODE REVIEW

File: DataView.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  letter-spacing: 1px;
+  background-color: #17a2b8;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  padding: 5px 15px;
+  margin: 10px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+  margin-top: 10px;
```
Comment (aaminbaig) [inline DataView.css]: you've already used **margin: 10px** not sure if this is still needed?

File: Signup.js
Comment (aaminbaig) [inline Signup.js:1]: Your PR shouldn't have irrelevant changes. 

Sign Up changes shouldn't be a part of this PR, makes it difficult for the reviewer to differentiate between the actual task changes and some other random changes, that have nothing to do with the PR.

File: DataView.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+document.addEventListener("DOMContentLoaded", () => {
+  fetch("https://jsonplaceholder.typicode.com/posts")
+    .then((response) => response.json())
+    .then((data) => {
+      const tableBody = document.getElementById("data-table-body");
+
+      const first10Posts = data.slice(0, 10);
```
Comment (aaminbaig) [inline DataView.js:7]: Is there a specific reason to slice the data?

**Review by aaminbaig — CHANGES_REQUESTED** [review]

File: DataView.js (hunk 2 above)
Comment (zarafshanejaz786) [reply DataView.js:7]: yes because there was a large number of data in response. that's why I sliced it.

File: Signup.js
Comment (zarafshanejaz786) [reply Signup.js:1]: Sorry for the inconvenience, actually my task 6 was not yet approved by the review that's why I couldn't merge it in the main.

File: DataView.css (hunk 1 above)
Comment (zarafshanejaz786) [reply DataView.css]: removed, this was mistakenly added, while I was pasting classes from my other style sheets.

**Review by aaminbaig — APPROVED** [review]

**Pull request merged**